app/assets/stylesheets/forms.scss

```
@@ -0,0 +1,118 @@
+// Form Elements
+// -------------
+// Inputs
+// Textarea
+// Checkboxes
+// Buttons
+
+fieldset {
+  padding: 0;
+  border: none;
+  margin-bottom: 24px;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+  }
+}
+
+legend {
+  color: #202428; // smoke-90
+  font-weight: 500;
+}
+
+label,
+input,
+textarea {
+  box-sizing: border-box;
+  margin: 0;
+  width: 100%;
+}
+
+// Input Fields
+label {
+  color: #202428; // smoke-90
+  display: flex;
+  flex-direction: column;
+  font-weight: 500; // medium
```

```suggestion
  font-weight: 500; // medium
```
i'm assuming that the comments in this file all the corresponding variables that we will be able to switch to once we have 🖍s?

---

Yup! I've checked in with @ludwiczakpawel , and we both said that the code in this PR won't be the best without the variables and components that Crayon will be giving us. But Crayon won't be in the codebase until about a few weeks, so we are comfortable including these notes for now.